Code/Tools/ProjectManager/Source/GemCatalog/GemInfo.cpp

```diff
@@ -100,4 +100,56 @@ namespace O3DE::ProjectManager
     {
         return (m_displayName < gemInfo.m_displayName);
     }
+
+    GemInfo::Platforms GemInfo::GetPlatformFromString(const QString& platformText)
+    {
+        if(platformText == "Windows")
+        {
+            return GemInfo::Platform::Windows;
+        }
+        else if(platformText == "Linux")
+        {
+            return GemInfo::Platform::Linux;
+        }
+        else if(platformText == "Android")
+        {
+            return GemInfo::Platform::Android;
+        }
+        else if(platformText == "iOS")
+        {
+            return GemInfo::Platform::iOS;
+        }
+        else if(platformText == "macOS")
+        {
+            return GemInfo::Platform::macOS;
+        }
+        else
+        {
+            return GemInfo::Platforms();
```

Comment:
In a later PR we should make sure we have some fallback for unknown platforms and not use the default which could be any value.

Reply:
will file github issue for this.